### PR TITLE
#2240 Update README

### LINKED ISSUE #2240: Update repo README
Do grammar/formatting corrections.

### PR DESCRIPTION
**Pull request by vahram-papazyan**
This PR makes grammar/formatting/content updates to the repo README

close #2240 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
 brew install swiftformat
-# install dependencies
+# 5. Install the dependencies
 make dependencies
 ```
 
-## Development
+## 🖥️ Development
+
+We recommended developing and running tests using only the `Debug FlowCrypt` scheme.
```
Comment (sosnovsky) [inline README.md:33]: `We recommended` -> `We recommend`

File: README.md [hunk 2]
```diff
@@ -1,51 +1,56 @@
...
-1) get uuid of their device and input it into https://developer.apple.com/account/ -> Devices
-2) choose the right build (eg `FlowCrypt Enterprise`) and choose `Any iOS Device (arm64)`
-3) Xcode -> Product -> Archive
-4) Distribute app -> Add Hoc -> Next -> Next (automatically manage signing)
-5) This creates a folder at the target where you export it to, and the IPA will be there
+1\. Get the UUID of their device and input it into the [Apple Developer](https://developer.apple.com/account/) account &#10140; **Devices**.
+2\. Choose the right build (e.g. **FlowCrypt Enterprise**) and choose **Any iOS Device (arm64)**.
+3\. **Xcode** &#10140; **Product** &#10140; **Archive**.
+4\. **Distribute app** &#10140; **Add Hoc** &#10140; **Next** &#10140; **Next** (automatically manage signing).
+5\. This creates a folder at the target where you export it, and the IPA will be there.
```
Comment (sosnovsky) [inline README.md:56]: It's currently rendered as a single line

Screenshot 2023-06-15 at 11 26 57

**Review by sosnovsky — COMMENTED** [review]
Nice icons 👍

File: README.md [hunk 3]
```diff
@@ -1,51 +1,56 @@
-# flowcrypt-ios
+# FlowCrypt iOS App (Encrypt email with PGP)
 
 ![Build Status](https://flowcrypt.semaphoreci.com/badges/flowcrypt-ios.svg?key=9bd38bf4-4a38-4cb3-b551-38302af1eb07)
 
-FlowCrypt iOS App, download from https://flowcrypt.com/download
+| &#8678; Prerequisites                                                                                          |
+|:---------------------------------------------------------------------------------------------------------------|
+| Download the FlowCrypt app on your device from the [FlowCrypt Downloads](https://flowcrypt.com/download) page. |
```
[the remark this answers is not in the document]
Comment (vahram-papazyan) [reply README.md:7]: I mean try the app to get familiar wit the product you're working on. I'll change the text with "Try our app" to be more precise

File: README.md (hunk 1 above)
Comment (vahram-papazyan) [reply README.md:33]: \+

File: README.md (hunk 2 above)
Comment (vahram-papazyan) [reply README.md]: \+

File: README.md [hunk 4]
```diff
@@ -1,51 +1,56 @@
-# flowcrypt-ios
+# 📱 FlowCrypt iOS App (🛡️ Encrypt email with PGP)
```
Comment (sosnovsky) [inline README.md:1]: let's remove 🛡️ icon

**Review by sosnovsky — COMMENTED** [review]
Looks much better now 👍

File: README.md (hunk 4 above)
Comment (vahram-papazyan) [reply README.md]: \+

**Review by sosnovsky — DISMISSED** [review]

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**